lib/dest/index.js

```diff
@@ -12,11 +12,21 @@ function dest(outFolder, opt) {
   }
 
   function saveFile(file, enc, cb) {
+    /* jshint validthis: true */
+    var stream = this;
+
+    function complete(err, file) {
+      cb(err, file);
+      if (!err) {
```

Comment:
the return value from push isn't a done flag (or that's what I think I found in all my digging).  I tried implementing your suggestion in #120 but it didn't seem to work.  The `cb` is being called on the line above with the `err` and `file` objects.  If an error occurs and is passed to that, the `error` event should be emitted on the stream and the pipeline torn down.  I check to see if there wasn't an error before reading because I don't know if reading from a torn down stream blows up or not.